pkg/registry/apps/daemonset/storage/storage.go

```diff
@@ -96,6 +96,6 @@ func (r *StatusREST) Get(ctx context.Context, name string, options *metav1.GetOp
 }
 
 // Update alters the status subset of an object.
-func (r *StatusREST) Update(ctx context.Context, name string, objInfo rest.UpdatedObjectInfo, createValidation rest.ValidateObjectFunc, updateValidation rest.ValidateObjectUpdateFunc) (runtime.Object, bool, error) {
-	return r.store.Update(ctx, name, objInfo, createValidation, updateValidation)
+func (r *StatusREST) Update(ctx context.Context, name string, objInfo rest.UpdatedObjectInfo, createValidation rest.ValidateObjectFunc, updateValidation rest.ValidateObjectUpdateFunc, forceAllowCreate bool) (runtime.Object, bool, error) {
+	return r.store.Update(ctx, name, objInfo, createValidation, updateValidation, forceAllowCreate)
```

Comment:
for subresources, I thought we weren't allowing create-on-update. doesn't that mean subresource rest storage should always propagate `false` to it's underlying store?

Reply:
Which would mean to ignore the bool. Shouldn't this be an error instead?

Reply:
I think propagating false makes sense in this case. we can't guarantee that all resources with custom storage implementations will have the ability to create on update, in those cases it would probably be better to just not honor the flag, so they can still be updated.

Reply:
I'll change this one, and make sure no other subresources are doing it this way

Reply:
@liggitt 
Okay, fixed this